tests/foreman/installer/test_infoblox.py

```diff
@@ -2,11 +2,9 @@
 
 :Requirement: Infoblox
 
-:CaseAutomation: Automated
-
 :CaseLevel: System
 
-:CaseComponent: Installer
+:CaseComponent: Infobloxintegration
```

Comment:
@pgagne Is this bugzilla component? I am not sure we are strictly bound to bz components or not. @mirzal am I right?

Reply:
It is 

https://bugzilla.redhat.com/describecomponents.cgi?product=Red%20Hat%20Satellite%206

Reply:
@pgagne Thanks. I was looking in mojo doc. We might need to update there.